src/DotNetWorker/DotNetWorker.csproj

```diff
@@ -9,7 +9,7 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <MinorProductVersion>12</MinorProductVersion>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <VersionSuffix>-preview2</VersionSuffix>
```

Comment:
For this PR, I am only bumping only the preview versions. We are exposing a 2 new types to public. So can bump up minor version when we are ready to publish the packages. Or we could include this in the non preview version of the latest package.